src/ziggurat/init.clj

```diff
@@ -102,17 +100,14 @@
   (stop-kafka-producers))
 
 (defn stop-server []
-  (mount/stop #'server/server)
-  (stop-rabbitmq-connection))
+  (mount/stop #'server/server))
 
 (defn stop-stream []
   (stop-kafka-streams)
-  (stop-rabbitmq-connection)
   (stop-kafka-producers))
 
 (defn stop-management-apis []
-  (mount/stop #'server/server)
-  (stop-rabbitmq-connection))
+  (mount/stop #'server/server))
```

Comment:
<img width="974" alt="Screenshot 2022-09-12 at 4 57 04 PM" src="https://user-images.githubusercontent.com/13599208/189642496-153a2d00-50b7-4642-b8d4-3f253d65d66d.png">
This adjustment is done to ensure subscribers stop before channel pool/connections.